### PR TITLE
Go formatted some stuff.

### PR DESCRIPTION
**Pull request by cparratto**

### CODE REVIEW

File: openstack/identity/v3/projects/requests.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package projects
+
+import (
+	"github.com/rackspace/gophercloud"
+)
+
+type response struct {
```
Comment (ghost) [inline openstack/identity/v3/projects/requests.go:7]: unused

File: openstack/identity/v3/projects/results.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package projects
+
+import (
+	"github.com/mitchellh/mapstructure"
```
Comment (ghost) [inline openstack/identity/v3/projects/results.go:4]: is this import already used by gophercloud, if not they might not be happy that you're forcing another dependency on them

File: openstack/identity/v3/projects/requests.go [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	IsDomain    bool   `json:"is_domain,omitempty"`
+	Description string `json:"description,omitempty"`
+	DomainID    string `json:"domain_id,omitempty"`
+	Enabled     bool   `json:"enabled,omitempty"`
+	Name        string `json:"name"`
+	ParentID    string `json:"parent_id,omitempty"`
+}
+
+// Create adds a new project using the provieded client.
+func Create(client *gophercloud.ServiceClient, opts *CreateOpts) CreateResult {
```
Comment (ghost) [inline openstack/identity/v3/projects/requests.go]: should `opts` really be a pointer ?

File: openstack/identity/v3/projects/urls.go [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package projects
```
Comment (ghost) [inline openstack/identity/v3/projects/urls.go:1]: 1 file, 7 lines, for a function called once, not sure

File: openstack/identity/v3/projects/results.go (hunk 2 above)
Comment (cparratto) [reply openstack/identity/v3/projects/results.go:4]: they use it already

File: openstack/identity/v3/projects/requests.go (hunk 3 above)
Comment (cparratto) [reply openstack/identity/v3/projects/requests.go]: does not need to be no

File: openstack/identity/v3/projects/urls.go (hunk 4 above)
Comment (cparratto) [reply openstack/identity/v3/projects/urls.go:1]: This is how they do it and my plan is to push it back up so I need to follow their conventions.

File: openstack/identity/v3/projects/requests_test.go [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+		    "enabled": true,
+		    "is_domain": true,
+		    "name": "myNewProject"
+		  }
+		}`)
+
+		w.Header().Add("Content-Type", "application/json")
+		w.WriteHeader(http.StatusCreated)
+		fmt.Fprintf(w, `{
+            "project": {
```
Comment (vichoward) [inline openstack/identity/v3/projects/requests_test.go:32]: is this the standard indentation under the fmt line or should it be indented further?  kindof a nit sorry ;).

**Review by vichoward — COMMENTED** [review]
LGTM, one comment

**Pull request merged**